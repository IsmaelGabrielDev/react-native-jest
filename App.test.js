import { expect } from '@jest/globals'
import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

test('Tela de renderização do App React Native', () => {
    const app = renderer.create(<App />).toJSON();
    expect(app).toMatchSnapshot();
});