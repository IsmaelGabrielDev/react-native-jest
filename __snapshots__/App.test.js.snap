// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tela de renderização do App React Native 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Meu primeiro App React Native na EBAC
  </Text>
</View>
`;
